pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -33,36 +32,24 @@ import (
 )
 
 type ConfigurationEditFlags struct {
+	//Fields for PodSpec
+	PodSpec knflags.PodFlags
```

Comment:
Mayeb call the field `PodSpecFlags`, too, to avoid confusion with a "real" PodSpec ?

(the type could be also called `PodSpecFlags` as it e.g. does not enclose the Pod template's meta data like labels or annotations)

Reply:
I changed to PodSpecFlags in the new version. Thank you.